docs/webhooks.rst

```diff
@@ -40,6 +40,17 @@ your docs whenever you push updates:
 If you ever need to manually set the webhook on Bitbucket,
 you can point it at ``https://readthedocs.org/bitbucket``.
 
+GitLab
+------
+
+If your project is hosted on GitLab, you can manually set the webhook on Gitlab and
```

Comment:
GitLab branding is to always call it `GitLab.com` (though I can't find their public documentation on that at this time).

However it's usually referred to as `GitLab.